pages/common/youtube-dl.md

```diff
@@ -6,6 +6,10 @@
 
 `youtube-dl {{https://www.youtube.com/watch?v=oHg5SJYRHA0}}`
 
+- Download the specific quality video or playlist:
```

Comment:
```suggestion
- Download a video or playlist at a specific quality:
```

(Although perhaps we should explicitly name this as "at the best available quality"? Which also leads me to wonder -- how is this particular example different from the default?)